src/app/utils/remove-duplicate.ts

```diff
@@ -0,0 +1,17 @@
+import { ApiResponseResult } from '../models/api-response';
+
+export function removeDuplicate(feeds: ApiResponseResult[], testFeeds: ApiResponseResult[]): ApiResponseResult[] {
```

Comment:
@Achint08 this should not be here, the utils functions, are the functions, which are generic, to all array types, make a generic function, which doesn't depend on **ApiResponseResult** then use it.